autoload/fzf/vim.vim

```diff
@@ -30,6 +30,9 @@ set cpo&vim
 
 let s:min_version = '0.23.0'
 let s:is_win = has('win32') || has('win64')
+if s:is_win
+  let s:is_wsl_bash = exepath('bash') =~? 'Windows[/\\]system32[/\\]bash.exe$'
+endif
```

Comment:
Can you change this to `let s:is_wsl_bash = s:is_win && ...` so that the variable is always defined?

Reply:
I certainly can. Thanks for the suggestion!